File: v2/java_sample/payloads/EnvelopeFieldsPayload.java
package payloads;

import org.json.JSONObject;

public class EnvelopeFieldsPayload extends FieldsPayload {

	private int recipient_id;
	private int original_id;

	public EnvelopeFieldsPayload(String email, String type, int page_number, double x, double y, double height,
			double width,int recipient_id,int original_id) {
		super(email, type, page_number, x, y, height, width);
		this.recipient_id = recipient_id;
		this.original_id = original_id;
	}
	
	public JSONObject toJson() 
	{
		JSONObject obj = new JSONObject();
		obj = super.toJson();
		obj.put("recipient_id", this.recipient_id);
		obj.put("original_id", this.original_id);
		
		return obj;
	}

}

File: v2/node_sample/payload/EnvelopeFieldsPayload.js
const FieldsPayload = require('./FieldsPayload');

class EnvelopeFieldsPayload extends FieldsPayload {
	constructor(email, type, page_number, x, y, height, width, recipient_id, original_id,
	required=false, additional_info={})
	{
	    super(email, type, page_number, x, y, height, width, required=false, additional_info={})
		this.recipient_id = recipient_id;
		this.original_id = original_id;
	}

	to_json()
	{
	    var obj = super.to_json();
		obj["recipient_id"] = this.recipient_id;
		obj["original_id"] = this.original_id;
		return obj
	}
}

module.exports = EnvelopeFieldsPayload